utils_test.py

```diff
@@ -0,0 +1,20 @@
+import os
+import unittest
+import utils
```

Comment:
Please arrange imports in descending order of line length, and separate the built-in from our own imports by a newline (our owns come after).

Plus, move this into `test/test_utils.py`